src/components/AddBookButton.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import Modal from 'react-modal'
+import FontAwesome from 'react-fontawesome'
+import AddBookForm from './AddBookForm.jsx'
+import ModalWindow from './ModalWindow.jsx'
+
+export default class AddBookButton extends React.Component {
+
```

Comment:
no extra lines here, generally no space between component declaration and constructor